views/layouts/main.hbs

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <title>Shopping Site</title>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="Page Layout">
+    <link rel="stylesheet" type='text/css' href="/main.css" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" type='text/css' href='https://fonts.googleapis.com/css?family=Dosis:400,600,700'
+        rel='stylesheet' />
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.0/css/bootstrap.min.css">

```

Comment:
please dont use it

Reply:
<link rel="stylesheet" type='text/css' href='https://fonts.googleapis.com/css?family=Dosis:400,600,700'
        rel='stylesheet' />

this was there in the document to use this font family